src/convertToObject.js

```diff
@@ -3,16 +3,24 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ *  properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';')
+    .map(element => element.split(':')).filter(element => element.length === 2)
+    .reduce((previous, current) => {
+      previous[current[0].trim()] = current[1].trim();
```

Comment:
All chained methods called on the same level should have same indentation.
```suggestion
  return sourceString
    .split(';')
    .map(element => element.split(':'))
    .filter(element => element.length === 2)
    .reduce((previous, current) => {
      previous[current[0].trim()] = current[1].trim();
```